deployer/src/config_manager.py

```diff
@@ -12,77 +14,123 @@
 
 
 class ConfigManager:
-    differ = None
+    instance = None
 
     def __init__(self):
-        self.ref_configs = fetchers.get_configs_from_repos()
-        self.actual_configs, self.inverted_actual_configs, self.crawler_ids = fetchers.get_configs_from_website()
+        if not ConfigManager.instance:
+            ConfigManager.instance = ConfigManager.__ConfigManager()
 
-        self.differ = DictDiffer(self.ref_configs, self.actual_configs)
+    class __ConfigManager:
+        def __init__(self):
+            self.public_dir = environ.get('PUBLIC_CONFIG_FOLDER')
+            self.private_dir = environ.get('PRIVATE_CONFIG_FOLDER')
```

Comment:
Could we have protection checks on these env?  Or at least a fallback

Reply:
The check is just under

Reply:
Imo it's ok. Since the command is only internal and the error is explicit.